app/javascript/components/patient/close_contact/CloseContact.js

```diff
@@ -42,32 +44,31 @@ class CloseContact extends React.Component {
       : 'enter additional information about monitoree’s potential exposure';
   }
 
-  toggleModal = () => {
-    let newState = {
-      showModal: !this.state.showModal,
-    };
-    if (this.state.showModal) {
-      // if we currently are showing the modal, that means they clicked cancel
-      // (because the rest of this method hasnt had time to fire, and change 'show' to false)
-      // When they click cancel, we want to null out all of the fields
-      newState = {
-        disableCreate:
-          (!this.props.close_contact.first_name && !this.props.close_contact.last_name) ||
-          (!this.props.close_contact.primary_telephone && !this.props.close_contact.email),
-        errors: {},
-        first_name: this.props.close_contact.first_name || '',
-        last_name: this.props.close_contact.last_name || '',
-        primary_telephone: this.props.close_contact.primary_telephone || '',
-        email: this.props.close_contact.email || '',
-        last_date_of_exposure: this.props.close_contact.last_date_of_exposure || null,
-        assigned_user: this.props.close_contact.assigned_user || null,
-        notes: this.props.close_contact.notes || '',
-        enrolled_id: this.props.close_contact.enrolled_id || null,
-        contact_attempts: this.props.close_contact.contact_attempts || 0,
-        ...newState, // merge in the flip-flopped value of showModal
-      };
-    }
-    this.setState(newState);
+  toggleDeleteModal = () => {
```

Comment:
can you move the delete methods below `toggleCloseContactModal` and `handleCloseContactSubmit`? the change handlers should be at the top

Reply:
this still needs to be done